simple_parsing/helpers/serialization/decoding.py

```diff
@@ -11,7 +11,7 @@
 from functools import lru_cache, partial
 from logging import getLogger
 from pathlib import Path
-from typing import Any, Callable, TypeVar
+from typing import Any, Callable, Literal, TypeVar
```

Comment:
This is what's causing an issue: It's not in `typing` yet for Python 3.7.
```suggestion
from typing import Any, Callable, TypeVar
from typing_extensions import Literal
```